crates/wdl-engine/src/stdlib/read_json.rs

```diff
@@ -156,18 +181,23 @@ mod test {
                 .collect::<Vec<_>>(),
             [1, 2, 3]
         );
+    }
 
-        let diagnostic = eval_v1_expr(&env, V1::One, "read_json('bad_array.json')")
+    #[tokio::test]
+    async fn read_bad_array_json() {
+        let diagnostic = eval_v1_expr(&make_env(), V1::One, "read_json('bad_array.json')")
             .await
             .unwrap_err();
         assert_eq!(
             diagnostic.message(),
-            "call to function `read_json` failed: failed to deserialize JSON file \
-             `bad_array.json`: a common element type does not exist between `Int` and `String` at \
-             line 1 column 11"
```

Comment:
This is the one place where it's revealed that switching away from doing these conversions via `serde` gives errors slightly less information about where a type error has occurred. Production code paths from `sprocket` already are converting first to `serde_json::Value` and then to `wdl_engine::Value` in discrete steps, however, so this does not constitute a regression to the quality of user-facing errors.

It might be nice to eventually add such location information to user-facing errors, but that should be treated as a future feature addition rather than as a reason to keep unnecessary complications in existing code.

Reply:
Ah, you're right, I had looked at the lack of regressions in integration tests involving `read_json` and figured that meant that this error wasn't bubbling up there either. I'll change it back 👍🏻